scripts/lib/api/api-html-artifacts.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
I think this file is probably better at `scripts/api-html-artifacts.json`.  It is not normal library code. We expect you to change this file! We want it to be more easily discoverable and make it clear it's not normal helper code.

Reply:
Done!